src/lib/components/simple/common/Divider.svelte

```diff
@@ -0,0 +1,19 @@
+<script lang="ts">
+	export let color = 'rgb(var(--global-background-color-500))',
```

Comment:
non passiamo mai le variabili css come javascript, metterle in un file css a parte e fare i vari default